vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -115,6 +115,12 @@ func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, recommendation *vpa
 			return
 		}
 	}
+
+	// If the pod has quick OOMed then evict only if the resources will change
+	if quickOOM && updatePriority.resourceDiff == 0 {
+		klog.V(2).Infof("not updating pod %v because resource would not change", pod.Name)
```

Comment:
nit: not updating pod %v/%v because resources would not change", pod.Namespace, pod.Name